pro/internet-gateways.rst

```diff
@@ -0,0 +1,75 @@
+===================================
+Internet Gateways
+===================================
+
+Internet gateways is a Pro-only feature that allows hosts in a Netmaker mesh network to connect to the internet via a gateway. This brings the possibility of a traditional VPN to machines with netclient installed on them. Internet gateways can be configured from the Internet gateway tab.
+
+.. image:: images/inet-gw-feature.png
+   :alt: internet gateway feature
+   :align: center
+   :width: 50%
+
+Only Linux hosts can be setup as internet gateways. Both Windows and Linux hosts can be connected to an internet gateway however. Remote clients can also be connected to internet gateways via a client config.
+
+This feature was introduced in v0.23.0 and is in beta (for this release).
+
+Any relay behind an internet gateway will have its traffic, as well as traffic of its relayed hosts going through the internet gateway.
+This behaviour also happens with remote access gateways and their connected clients; a remote access gateway behind an internet gateway would have its traffic, as well as traffic of its clients, passing through the internet gateway.
+
+
+**For Community Edition (CE) Users**, clients can still access internet gateways via the `Remote Access tab`. View `here for more info <../external-clients.html#internet-gateway-traditional-vpn>`_.
```

Comment:
can we add this prompt for community under remote access gateway? if this is under pro section, an user using community will never see this